packages/pymedphys_analysis/src/pymedphys_analysis/gamma/_level1/gammafilter.py

```diff
@@ -44,17 +46,17 @@ def calculate_pass_rate(gamma_array):
     return percent_pass
 
 
-def gamma_filter_brute_force(coords_reference, dose_reference,
-                             coords_evaluation, dose_evaluation,
+def gamma_filter_brute_force(axes_reference, dose_reference,
+                             axes_evaluation, dose_evaluation,
                              distance_mm_threshold, dose_threshold,
                              lower_dose_cutoff=0, **kwargs):
 
-    xx_ref, yy_ref, zz_ref = np.meshgrid(*coords_reference, indexing='ij')
+    coords = coords_from_xyz_axes(axes_reference)
```

Comment:
I don't believe I ever wrote tests for gamma filter. Did I? I'm just doing this on my mobile... Is gamma filter still accurate with the axis labels being placed on different array dimensions?

Reply:
I should've been more careful here. Let's remove that change for now.

Reply:
Kk, so this probably the only change I need and then I'm all good for this to merge. :)